packages/python/accern_xyme/accern_xyme.py

```
@@ -435,11 +437,11 @@ def check_error(req: Response) -> None:
         if method == METHOD_GET:
             req = requests.get(url, params=args, headers=headers)
             check_error(req)
-            return StringIO(req.text)
+            return StringIO(f"{req.text}")
```

does this hide errors?

---

reverted